<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ib_includeStation</class>
 <widget class="QWidget" name="ib_includeStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="includestation_bar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#changepassword_bar {
    border-style:solid;
	border-width:1px;
	border-color:rgb(187, 187, 187);
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(242, 242, 242), stop:1 rgb(221, 221, 221));
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_icon">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/Images/logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_bar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="station_bar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="stationname" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <widget class="QLabel" name="lb_stationname">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>10</y>
              <width>61</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>站点名称</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="le_stationname">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>10</y>
              <width>130</width>
              <height>30</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="location" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <widget class="QLabel" name="lb_lacation">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>10</y>
              <width>61</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>所在区域</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="le_location">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>10</y>
              <width>130</width>
              <height>30</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="street" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <widget class="QLabel" name="lb_street">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>10</y>
              <width>61</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>所在街道</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="le_street">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>10</y>
              <width>130</width>
              <height>30</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_searchstation">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_confirm">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DragWidget" name="listWidget_result">
        <property name="minimumSize">
         <size>
          <width>780</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#listWidget_result {
	border-width:1px;
	border-style:solid;
    border-radius: 2px;
    border-color: rgb(187, 187, 187);
}

QListWidget#listWidget_result::Item { 
	width: 100px; 
	height: 40px;
	background-color:white; 
	color:black;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(187, 187, 187);
	border-radius: 2px;
}
QListWidget#listWidget_result::Item:hover { 
	background-color: rgb(250, 187, 97); 
	color: white;
}
QListWidget#listWidget_result::item:selected { 
	background-color: rgb(249, 160, 34); 
	color: white;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DragWidget" name="listWidget_modify">
        <property name="minimumSize">
         <size>
          <width>780</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>780</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#listWidget_modify {
	border-width:1px;
	border-style:solid;
    border-radius: 2px;
    border-color: rgb(187, 187, 187);
}

QListWidget#listWidget_modify::Item { 
	width: 100px; 
	height: 40px;
	background-color:rgb(89, 179, 89); 
	color:white;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(187, 187, 187);
	border-radius: 2px;
}

QListWidget#listWidget_modify::Item:hover { 
	background-color: rgb(250, 187, 99); 
	color: white;
}

QListWidget#listWidget_modify::item:selected { 
	background-color: rgb(249, 160, 34); 
	color: white;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragWidget</class>
   <extends>QListWidget</extends>
   <header location="global">dragwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
